static/src/stylesheets/amp/_adconsent.scss

```diff
@@ -0,0 +1,35 @@
+amp-consent { background: none; }
+div.consent-ui {
+    font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+    font-size: 1rem;
+    line-height: 1.25rem;
+
+    color: #ffffff;
+    background-color: rgb(51, 51, 51);
+
+    a {
+        color: #ffffff;
+    }
+    a:hover {
+        text-decoration: 0;
+    }
+
+    div {
+        margin: 1em;
+    }
+    div.adconsent-actions {
+        text-align: right;
+        button {
+            border: 0;
+            background: transparent;
+            font: inherit;
+            color: inherit;
+        }
+        button.adconsent-accept {
```

Comment:
I believe we can use sass mixins and variables in the amp styles? 
Could we do something more similar to the `site-message--first-pv-consent__button`? Like this:

```css
.adconsent-accept {
    @include circular;
    height: $btn-height;
    background: $highlight-main;
    color: $brightness-7;
    ....
```

Reply:
I now use some available variables :)